### PR TITLE
Smaato: Change server response type

### PR DESCRIPTION
**Pull request by Enigo**
Based on the other branch

### CODE REVIEW

File: adapters/smaato/banner.go [hunk 1]
```diff
@@ -0,0 +1,32 @@
+package smaato
+
+import (
+	"fmt"
+	"net/url"
+	"strings"
+)
+
+type imageAd struct {
```
Comment (el-chuck) [inline adapters/smaato/banner.go]: do we use those types somewhere? i think they can go

File: adapters/smaato/banner.go [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+}
+
+func extractAdmBanner(adMarkup string, curls []string) string {
+	var clickEvent strings.Builder
+	for _, clicktracker := range curls {
+		clickEvent.WriteString("fetch(decodeURIComponent('" + url.QueryEscape(clicktracker) + "'.replace(/\\+/g, ' ')), " +
+			"{cache: 'no-cache'});")
+	}
+
+	return fmt.Sprintf(`<div style="cursor:pointer" %s>%s</div>`, clickEvent.String(), adMarkup)
```
Comment (el-chuck) [inline adapters/smaato/banner.go]: this will generate invalid html, there should be onclick before placing the trackers

Comment (Enigo) [reply adapters/smaato/banner.go]: not sure how I missed that, fixed now

**Review by el-chuck — APPROVED** [review]
lgtm

**Pull request closed without merging**